plugins/obs-widget/back.js

```diff
@@ -0,0 +1,56 @@
+const {ipcMain} = require("electron");
+const http = require("http");
+const registerCallback = require("../../providers/song-info");
+
+let currentSongInfo;
+const data = {
+	player: {
+		hasSong: false,
+		isPaused: true,
+		seekbarCurrentPosition: 0,
+	},
+	track: {
+		author: "",
+		title: "",
+		album: "",
+		cover: "",
+		duration: 0,
+	}
+};
+
+/** @param {Electron.BrowserWindow} win */
+module.exports = async (win) => {
+	const requestListener = function (req, res) {
+		if(req.url !== '/query'){
+			res.writeHead(404);
+			res.end("404 Not found!");
+		}
+		res.setHeader("Access-Control-Allow-Origin", "*");
+		res.writeHead(200);
+		if (currentSongInfo !== undefined && !currentSongInfo.title && !currentSongInfo.artist) {
+			res.end(JSON.stringify({}));
+			return;
+		}
+		data.player.hasSong = true;
+		data.player.isPaused = currentSongInfo.isPaused;
+		data.player.seekbarCurrentPosition = currentSongInfo.elapsedSeconds;
+		data.track.author = [currentSongInfo.artist];
+		data.track.title = currentSongInfo.title;
+		data.track.album = currentSongInfo.album;
+		data.track.cover = currentSongInfo.imageSrc;
+		data.track.duration = currentSongInfo.songDuration;
```

Comment:
Just a small lifehack :^)

```suggestion
                Object.assign(data, {
                    player: {
                        hasSong: true,
                        isPaused: currentSongInfo.isPaused,
                        seekbarCurrentPosition: currentSongInfo.elapsedSeconds
                    },
                    track: {
                        author: [currentSongInfo.artist],
                        title: currentSongInfo.title,
                        album: currentSongInfo.album,
                        cover: currentSongInfo.cover,
                        duration: currentSongInfo.duration
                    }
                });
```